### PR TITLE
Added service account page

### PR DESCRIPTION
**Pull request by cb1kenobi**
### DO NOT PUBLISH UNTIL AXWAY CLI 3.0.0

JIRA: https://jira.axway.com/browse/APIGOV-20266

### CODE REVIEW

**Review by sallylanda6 — CHANGES_REQUESTED** [review]
I am recommending some copy edit changes for consistency and to use active instead of passive voice.

File: content/en/docs/authentication/service_accounts.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+axway service-account create --name <value> --public-key <path>
+
+axway service-account create --name <value> --secret <key>
+
+axway service-account create --name <value> --secret <key> --client-id <id> --desc "<description>" --role <role1> --role <role2>
+```
+
+If the service account name is not specified, then the command will interactively prompt for all values. If prompting is not available, then all required options must passed in at execution.
+
+The client id defaults to the specified service account name plus a uuid. You can override this an specify your own client id, however the client id cannot begin with "AASA", "CASA", "DOSA", or "MASA".
```
[the remark this answers is not in the document]
Comment (cb1kenobi) [reply content/en/docs/authentication/service_accounts.md:64]: Not sure I agree. The sentence has been negated and thus the negation of "and" is "or". Just want to confirm, should this definitely be "and"?

[the remark this answers is not in the document]
Comment (cb1kenobi) [reply content/en/docs/authentication/service_accounts.md:64]: These are very specific values and I don't want to confused "MASA" with "MASA.". I've made each of these keywords code blocks.

[the remark this answers is not in the document]
Comment (sallylanda6) [reply content/en/docs/authentication/service_accounts.md]: Not sure what your comment refers to. This is where I was saying to change an to and... "You can override this an specify your own client id". I thought inline comments would highlight the exact area but I guess it does not and I need to be more specific.

This sentence "If prompting is not available, then all required options must passed in at execution" the word "be" is missing. "If prompting is not available, then all required options must be passed in at execution."

[the remark this answers is not in the document]
Comment (sallylanda6) [reply content/en/docs/authentication/service_accounts.md]: Perfect!

[the remark this answers is not in the document]
Comment (cb1kenobi) [reply content/en/docs/authentication/service_accounts.md]: OH! "an"!!!

File: content/en/docs/authentication/service_accounts.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+### add-team
+
+Adds an existing team to a service account. When assigning a team, you must also specify the team role.
+
+```
+axway service-account add-team <client-id/name> <team-guid/name> <role>
+```
+
+You may specify the service account by client id or name as well as the team by guid or name. The available team roles can be discovered by running `axway service-account roles`.
```
Comment (sallylanda6) [inline content/en/docs/authentication/service_accounts.md:42]: I missed one... change to 
Run `axway service-account roles` to discover the available team roles.

**Review by sallylanda6 — CHANGES_REQUESTED** [review]
Sorry, I missed one. I think this is all.

**Review by sallylanda6 — APPROVED** [review]
Looks good!

**Pull request merged**